sample-plugin/src/test/kotlin/com/github/jackchen/plugin/sample/SamplePluginTest.kt

```diff
@@ -5,78 +5,137 @@ import com.github.jackchen.gradle.test.toolkit.ext.TestVersion
 import org.gradle.testkit.runner.TaskOutcome
 import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
-import java.io.File
 
+/**
+ * @author JackChen
+ */
 class SamplePluginTest : GradlePluginTest() {
 
-    private fun testProjectSetup(closure: GradlePluginTest.() -> Unit) {
-        val pluginVersion = System.getProperty("io.github.jackchen365.sample.version")
-        val testPackageName = "com.github.jackchen.plugin.test"
-        kotlinAndroidTemplate {
-            template {
-                `package` {
-                    packageName = testPackageName
-                }
-                plugins {
-                    id("io.github.jackchen365.sample").version(pluginVersion)
-                }
-                dependencies {
-                    implementation("androidx.core:core-ktx:1.7.0")
-                    implementation("androidx.appcompat:appcompat:1.4.1")
-                    implementation("io.github.jackchen365:sample-api:$pluginVersion")
-                }
-            }
-            project {
-                module("app") {
-                    sourceDir(testPackageName) {
-                        file("test.md") {
-                            """
+  private fun testProjectSetup(
+    isEnableDebug: Boolean = false,
+    closure: GradlePluginTest.() -> Unit
+  ) {
+    val pluginVersion = System.getProperty("io.github.jackchen365.sample.version")
+    val testPackageName = "com.github.jackchen.plugin.test"
+    kotlinAndroidTemplate {
+      template {
+        `package` {
+          packageName = testPackageName
+        }
+        plugins {
+          id("io.github.jackchen365.sample").version(pluginVersion)
+        }
+        dependencies {
+          implementation("androidx.core:core-ktx:1.7.0")
+          implementation("androidx.appcompat:appcompat:1.4.1")
+          implementation("io.github.jackchen365:sample-api:$pluginVersion")
+        }
+      }
+      project {
+        module("app") {
+          file("build.gradle.kts") {
+            """
+                            plugins {
```

Comment:
The code format is weird. You can try this one

```
"""
|sample {
|      enableDebug.set($isEnableDebug)
|}
""".tripMargin()
```

Reply:
Good advice!